ui/src/groups/LureInviteBlock.tsx

```diff
@@ -96,35 +103,27 @@ export default function LureInviteBlock({
       )}
       {enabled && enableAcked ? (
         <div className="relative flex flex-1 items-center space-x-2">
-          {url === '' && !good && (
-            <div className="flex w-full items-center space-x-2 rounded-lg border-2 border-blue-soft bg-blue-soft py-1 px-2 text-lg font-semibold leading-5 text-blue mix-blend-multiply dark:mix-blend-screen sm:text-base sm:leading-5">
+          {!checked && (
+            <div className="flex w-full flex-1 items-center space-x-2 rounded-lg border-2 border-blue-soft bg-blue-soft py-1 px-2 text-lg font-semibold leading-5 text-blue mix-blend-multiply dark:mix-blend-screen sm:text-base sm:leading-5">
               <LoadingSpinner className="h-4 w-4 text-blue" />
               <span className="font-semibold text-blue">
-                Generating link...
+                {url ? 'Verifying link...' : 'Generating link...'}
               </span>
             </div>
           )}
-          {url !== '' && !good && (
+          {url !== '' && checked && !good && (
             <>
-              <input
-                value="Verifying link..."
-                readOnly
-                className="flex w-full flex-1 rounded-lg border-2 border-gray-100 bg-gray-100 py-1 px-2 text-lg font-semibold  leading-5 text-gray-600 mix-blend-multiply focus:outline-none dark:mix-blend-screen sm:text-base sm:leading-5"
-              />
-              <LoadingSpinner />
-              <button
-                disabled={retryDisabled}
-                className="button"
-                onClick={() => {
-                  toggle(group?.meta || emptyMeta)();
-                  setTimeout(toggle(group?.meta || emptyMeta), 500);
-                }}
-              >
+              <div className="flex w-full flex-1 items-center space-x-2 rounded-lg border-2 border-blue-soft bg-blue-soft py-1 px-2 text-lg font-semibold leading-5 text-blue mix-blend-multiply dark:mix-blend-screen sm:text-base sm:leading-5">
+                <span className="font-semibold text-blue">
+                  Failed to successfully create link with lure service
```

Comment:
I think we can go with "Error creating link" here. "Lure" makes sense to us for internal terminology but we don't explain it anywhere (nor should we IMO)

Reply:
```suggestion
                  Error creating link
```